test/unit/spread.spec.ts

```diff
@@ -76,6 +76,7 @@ describe("in function call", () => {
         {
             [tstl.LuaTarget.Universal]: builder => builder.tap(expectLualibUnpack).expectToMatchJsResult(),
             [tstl.LuaTarget.LuaJIT]: builder => builder.tap(expectUnpack),
+            [tstl.LuaTarget.Lua50]: builder => builder.tap(expectUnpack).expectToMatchJsResult(),
```

Comment:
Maybe I'm overlooking something, but do any of these existing tests cover the arg literal you added?

Reply:
No, I think you're onto something. These tests have to be split into 5.0 and non-5.0 cases and the snapshots updated to expect `arg` for 5.0.

Reply:
Refactored the spread tests with [a5f39f6](https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/a5f39f617bae32741ded2f6e08c580f103ae4b94). Since @lolleko is working on bindings for 5.0, I'll keep the match JS result calls.